src/main/java/ui/DroneDeck.java

```diff
@@ -1,17 +1,31 @@
 package main.java.ui;
 
 import com.formdev.flatlaf.FlatDarkLaf;
+import com.formdev.flatlaf.FlatLaf;
 import main.java.ui.components.MainPanel;
 
 import javax.swing.*;
+import java.awt.*;
+import java.io.InputStream;
 import java.util.Objects;
 
 public class DroneDeck {
 
     public static void main(String[] args) {
         // Set up FlatLaf look and feel
+        FlatLaf.registerCustomDefaultsSource("main.java.ui.themes");
         FlatDarkLaf.setup();
 
+        // Load Google Font
+        try (InputStream is = DroneDeck.class.getResourceAsStream("/Lato-Bold.ttf")){
+            Font font = Font.createFont(Font.TRUETYPE_FONT, Objects.requireNonNull(is)).deriveFont(14f);
+            UIManager.put("defaultFont", font);
+        }
+        catch (Exception e) {
+            //noinspection CallToPrintStackTrace
+            e.printStackTrace();
```

Comment:
Just as a reminder we need to implement a solid logging solution. But for now this should be ok :)